handlers/src/unit-test/java/uk/gov/moj/sdt/handlers/WsCreateBulkRequestHandlerTest.java

```diff
@@ -0,0 +1,151 @@
+package uk.gov.moj.sdt.handlers;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.stubbing.Answer;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.moj.sdt.domain.api.IBulkSubmission;
+import uk.gov.moj.sdt.services.api.IBulkSubmissionService;
+import uk.gov.moj.sdt.transformers.api.ITransformer;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+import uk.gov.moj.sdt.utils.concurrent.InFlightMessage;
+import uk.gov.moj.sdt.utils.concurrent.api.IInFlightMessage;
+import uk.gov.moj.sdt.utils.mbeans.SdtMetricsMBean;
+import uk.gov.moj.sdt.utils.visitor.VisitableTreeWalker;
+import uk.gov.moj.sdt.validators.api.IBulkSubmissionValidator;
+import uk.gov.moj.sdt.validators.exception.CustomerNotFoundException;
+import uk.gov.moj.sdt.ws._2013.sdt.bulkrequestschema.BulkRequestType;
+import uk.gov.moj.sdt.ws._2013.sdt.bulkrequestschema.HeaderType;
+import uk.gov.moj.sdt.ws._2013.sdt.bulkresponseschema.BulkResponseType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class WsCreateBulkRequestHandlerTest extends AbstractSdtUnitTestBase {
+
+    @Mock
+    IBulkSubmissionService mockBulkSubmissionService;
+
+    @Mock
+    IBulkSubmissionValidator mockBulkSubmissionValidator;
+
+    @Mock
+    ITransformer<BulkRequestType, BulkResponseType, IBulkSubmission, IBulkSubmission> mockTransformer;
+
+    @Mock
+    Map<String, IInFlightMessage> mockConcurrencyMap;
+
+    @Mock
+    BulkRequestType mockBulkRequestType;
+
+    @Mock
+    IBulkSubmission mockBulkSubmission;
+
+    @Mock
+    BulkResponseType mockBulkResponseType;
+
+    @Mock
+    HeaderType mockHeader;
+
+    @Spy
+    WsCreateBulkRequestHandler wsCreateBulkRequestHandler = new WsCreateBulkRequestHandler(null, null, null);
+
+    long startingBulkSubmitCount;
+
+    long startingActiveBulkCustomerCount;
+
+    @BeforeEach
+    @Override
+    public void setUp() {
+        wsCreateBulkRequestHandler.setBulkSubmissionService(mockBulkSubmissionService);
+        wsCreateBulkRequestHandler.setBulkSubmissionValidator(mockBulkSubmissionValidator);
+        wsCreateBulkRequestHandler.setTransformer(mockTransformer);
+        wsCreateBulkRequestHandler.setConcurrencyMap(mockConcurrencyMap);
```

Comment:
wsCreateBulkRequestHandler = new WsCreateBulkRequestHandler(null, null, null);
Move this here and pass mocked objects in constructor